aging.csl

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" sort-separator="," demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Aging</title>
+    <id>http://www.zotero.org/styles/aging</id>
+    <link rel="self" href="http://www.zotero.org/styles/aging"/>
+    <link href="http://www.impactaging.com/forAuthors.html" rel="documentation"/>
+    <author>
+      <name>Jose L. Aguilar</name>
+      <email>joseluis0988@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <category field="biology"/>
+    <category field="medicine"/>
+    <updated>2016-05-26T23:21:51+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="collection-editor" form="long">
+        <single>Series editor</single>
+        <multiple>Series editors</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name delimiter-precedes-last="always" et-al-use-first="13" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```

Comment:
`et-al-use-first="13"`

Is this intentional? Or are the settings `et-al-min="31" et-al-use-first="30"` on `<bibliography>` the correct ones?